doozer/doozerlib/cli/images_konflux.py

```diff
@@ -71,9 +72,15 @@ async def run(self):
                 commit_message=self.message,
                 push=self.push)))
         results = await asyncio.gather(*tasks, return_exceptions=True)
-        failed = [r for r in results if isinstance(r, Exception)]
-        if failed:
-            raise DoozerFatalError(f"Failed to rebase images: {failed}")
+        failed_images = []
+        for index, result in enumerate(results):
+            if isinstance(result, Exception):
+                image_name = metas[index].distgit_key
+                failed_images.append(image_name)
+                LOGGER.error(f"Failed to rebase {image_name}")
+                traceback.print_exception(result)
```

Comment:
Is this really needed? Printing the stack trace to the logs makes them less readable, I think it should be done only if strictly needed.

Reply:
otherwise LGTM

Reply:
There's no debug.log we have here which we can refer to. Until we have that we have no way of knowing where exactly the error occured

Reply:
reverted traceback, merging with minimal changes